linode/manager
# fix: [M3-6499] - Entity links in notification menu/events page

## Description 📝
- Clicking on entity links in the notification menu no longer causes a hard refresh
  - Previously, the links were just `a` tags so they weren't connected with the react-router-dom. The fix was to use the `Link` component which required some refactoring to get `eventMessageGenerator` to return JSX instead of a string
- Fix backticked word display on the event page
- Fix fake entity links that shouldn't be links
- Remove extra space and quotation marks from support message link

## Preview 📷
| Before  | After   |
| ------- | ------- |
| ![event page before](https://github.com/linode/manager/assets/115299789/120695ef-f508-41d2-b06c-8181eac50f03) | ![event page after](https://github.com/linode/manager/assets/115299789/7e378fb1-d21b-48d1-9460-9a433072389d) |

## How to test 🧪
```
yarn test eventMessageGenerator
```

- Check various events in the notification menu and event page
- Ensure that previous known regressions are not present e.g.
  - #9280
  - #9360
